lua/cfc_parachutes/shared/sh_parachute.lua

```diff
@@ -121,14 +124,14 @@ function CFC_Parachute._ApplyChuteForces( ply, chute, mv )
     local vel = mv and mv:GetVelocity() or ply:GetVelocity()
     local velZ = vel[3]
 
-    if velZ > cvFallZVel then return end
-
     local timeMult = FrameTime()
 
     -- Modify velocity.
     vel = addHorizontalVel( ply, chute, vel, timeMult )
     velZ = velZ + ( cvFallZVel - velZ ) * cvFallLerp * timeMult
 
+    if velZ > cvFallZVel then return end -- do this after we check the horizontal vel
```

Comment:
is this a note? a todo? what is this comment doing for us

Reply:
Yeah the comment should be better